src/lib/router.js

```diff
@@ -71,9 +76,13 @@ async function swapContent(url) {
   // Remove the current #content element
   main.querySelector("#content").remove();
   // Swap in the new #content element
-  main.appendChild(page.querySelector("#content"));
+  main.appendChild(content);
   // Update the page title
   document.title = page.title;
+  // Focus on the first title (or fallback to content itself)
+  (content.querySelector("h1, h2, h3, h4, h6, h6") || content).focus();
```

Comment:
Wanted to mention that this won't do anything unless the headings or content have tabindex=-1. We could add the tabindex in our various page templates.

Reply:
Also noticed there are two h6s on line 83

Reply:
oops re: h6

I've added a `forceFocus` function which sets a tabindex imperatively to move the cursor. WDYT of that approach from an a11y point of view?